config/peerpods/podvm/Dockerfile.podvm-builder

```diff
@@ -31,4 +31,6 @@ RUN git clone ${CAA_SRC} -b ${CAA_REF} /src/cloud-api-adaptor
 
 ADD podvm-builder.sh /podvm-builder.sh
 
+COPY parse_hdr.py /src/cloud-api-adaptor/src/cloud-api-adaptor/podvm
```

Comment:
Maybe we will add a condition here to copy this file only for s390x arch?

or if it's allowed how about downloading this script from the official trustee repo while on the relevant shell function? so that we need to maintain the script in just one place?

Reply:
Actually this is our local Zaas Script and we can change it also as per further need so I have tried to keep it local.

Reply:
can you please check whether it's same as of https://github.com/confidential-containers/trustee/blob/main/deps/verifier/src/se/se_parse_hdr.py

Reply:
yeah , it looks same. I will try to take it and use rather than creation of new one

Reply:
Done. thanks @ajayvic for pointing it.